types/node-forge/cipher.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module "node-forge" {
+    namespace cipher {
+        type Algorithm =
+            | "AES-ECB"
+            | "AES-CBC"
+            | "AES-CFB"
+            | "AES-OFB"
+            | "AES-CTR"
+            | "AES-GCM"
+            | "3DES-ECB"
+            | "3DES-CBC"
+            | "DES-ECB"
+            | "DES-CBC";
+
+        function createCipher(algorithm: Algorithm, payload: util.ByteBuffer | Bytes): BlockCipher;
+        function createDecipher(algorithm: Algorithm, payload: util.ByteBuffer | Bytes): BlockCipher;
+
+        interface StartOptions {
+            iv?: util.ByteBuffer | Byte[] | Bytes | undefined;
+            tag?: util.ByteStringBuffer | undefined;
+            tagLength?: number | undefined;
+            additionalData?: string | undefined;
+        }
+
+        interface BlockCipher {
+            start: (options?: StartOptions) => void;
+            update: (input: util.ByteBuffer) => void;
```

Comment:
This is not a pure move.

Reply:
It's the same type, just changed parameter name to name from source.

Reply:
Great—but do it in a separate commit from the pure move.